pkg/k8s/synced/crd.go

```diff
@@ -105,7 +105,8 @@ func AllCiliumCRDResourceNames() []string {
 	return append(
 		AgentCRDResourceNames(),
 		CRDResourceName(v2.CEWName),
-		CRDResourceName(v2alpha1.CNCName),
+		CRDResourceName(v2.CNCName),
+		CRDResourceName(v2alpha1.CNCName), // TODO remove me when CNC v2alpha1 will be deprecated
```

Comment:
As mentioned above, let's file an issue to use that to reference the removal of the deprecated code in all of the TODO msgs.

Reply:
Fixed.